integration/spark/src/test/common/java/io/openlineage/spark/agent/lifecycle/LibraryTest.java

```diff
@@ -119,7 +119,7 @@ public void testRdd(SparkSession spark) throws IOException {
     List<OpenLineage.RunEvent> events = lineageEvent.getAllValues();
     assertEquals(2, events.size());
 
-    ObjectMapper objectMapper = OpenLineageClient.getObjectMapper();
+    ObjectMapper objectMapper = Utils.newObjectMapper();
```

Comment:
We already do have different kinds of `Utils` classes. 
If leaving `OpenLineageClient` with static method `getObjectMapper` and producer uri is not good enough (this would limit amount of files changed), consider more explicit class name like `EmitterUtils`, `OpenlineageClientUtils`, etc.

Reply:
I can rename the class in follow-up PR - this way the PR won't get even more complicated, ok?